CommunityFridgeMapApi/functions/email_service/v1/send_email.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import boto3
+from botocore.exceptions import ClientError
+
+client = boto3.client('ses', "us-east-1")
+
+CHARSET = "UTF-8"
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def sendEmail(sender, recipient, subject, body) -> int:
+    #   Try to send the email.
+    try:
+        # Provide the contents of the email.
+        response = client.send_email(
+            Source=sender,
+            Destination={
+                'ToAddresses': [
+                    recipient
+                ],
+            },
+            Message={
+                'Body': {
+                    'Html': {
```

Comment:
nit: Why are there 2 different fields (`html` and `text`) using the same data?

Reply:
Ya, i'm not sure how to handle the html. I'm not super familiar with email html handling? Does anyone know anything about that?

Reply:
This is what the documentation says:
* `
Text (dict) --
The content of the message, in text format. Use this for text-based email clients, or clients on high-latency networks (such as mobile devices).
`

* `
Html (dict) --
The content of the message, in HTML format. Use this for email clients that can process HTML. You can include clickable links, formatted text, and much more in an HTML message.
`

Looks like we can choose either or? I assume we would only need the text version for this email